src/api/api.model.ts

```diff
@@ -142,10 +142,46 @@ export class TokenPairWithLiquidityInfo {
 export class TokenWithUsd extends TokenWithListed {
   @ApiProperty({
     description: 'Price of the token in USD',
-    pattern: bigNumberPattern,
+    pattern: usdValuePattern,
+  })
+  priceUsd: number;
+
+  @ApiProperty({
+    description: 'Fully diluted valuation in USD',
+    pattern: usdValuePattern,
   })
-  priceUsd: string;
+  fdvUsd: number;
+
+  @ApiProperty({
+    description: 'Volume for last day in USD',
+    pattern: usdValuePattern,
+  })
+  volumeUsdDay: number;
+
+  @ApiProperty({
+    description: 'Volume for last day in USD',
```

Comment:
```suggestion
    description: 'Volume for last week in USD',
```
and the other ones as well

Reply:
done